src/state/league.js

```diff
@@ -175,6 +175,12 @@ const loadManualResultsFromSheets = async () => {
     if (row["Comment"]) {
       result.extraInfo = row["Comment"];
     }
+    if (row["Debutant"]) {
+      result.isDebutant = !!row["Debutant"];
+    }
+    if (row["DNF"]) {
+      result.isDnfEntry = !!row["DNF"];
```

Comment:
this is probably fine but for extra fine tuning possibilities you could go:
```
if (row["DNF"]) {
  if (row["DNF"]).toLowerCase() === "false") {
    result.isDnfEntry = false;
  } else {
    results.isDnfEntry = true;
  }
}

```
this is because the string "false" is truthy, so !!"false" = true :)